pkg/metrics/kubernetes_client.go

```diff
@@ -81,14 +82,14 @@ func init() {
 		Name:        "k8s_client_cache_list_duration_seconds",
 		Measure:     cacheListLatencyStats,
 		Description: "Duration of a Kubernetes list API call in seconds",
-		Aggregation: view.Distribution(0, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5, 0.75, 1, 2, 3),
+		Aggregation: view.Distribution(distributionSeconds...),
 	}))
 
 	runtime.Must(view.Register(&view.View{
 		Name:        "k8s_client_cache_list_items",
 		Measure:     cacheListItemCountStats,
 		Description: "Count of items in a list from the Kubernetes API.",
-		Aggregation: view.Distribution(0),
+		Aggregation: view.Distribution(0.00001),
```

Comment:
I wonder why past me didn't put any buckets here ?

I think we should have 0,10,50,100,150,250,300.